test/sycl_tests.cc

```diff
@@ -62,4 +62,90 @@ TEMPLATE_TEST_CASE_METHOD_SIG(dim_device_queue_fixture, "ranged_sycl_access work
 	CHECK(actual == expected);
 }
 
+
+template <access_mode, bool>
+class access_test_kernel;
+
+#if WORKAROUND_DPCPP || WORKAROUND(COMPUTECPP, 2, 7)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations" // target::gobal_buffer is now target::device, but only for very recent versions of DPC++
+constexpr auto sycl_target_device = cl::sycl::access::target::global_buffer;
+#pragma GCC diagnostic pop
+#else
+constexpr auto sycl_target_device = cl::sycl::access::target::device;
+#endif
+
+template <access_mode AccessMode, bool UsingPlaceholderAccessor>
+static auto make_device_accessor(sycl::buffer<int, 1>& buf, sycl::handler& cgh, const subrange<1>& sr) {
+	if constexpr(UsingPlaceholderAccessor) {
+		sycl::accessor<int, 1, AccessMode, sycl_target_device, sycl::access::placeholder::true_t> acc{buf, sr.range, sr.offset};
```

Comment:
I don't know if this was intentional since I assume not all implementations have caught up yet, but the `placeholder_t` template parameter is a no-op in SYCL 2020.

Reply:
That's duplicated from accessor.h lines 286 f. hipSYCL seems to be the only implementation to have removed `placeholder_t`.